client/src/components/pages/application/Signup/index.js

```diff
@@ -1,5 +1,156 @@
 import React from 'react';
+import axios from 'axios';
+import Card from '../../../common/card';
+import Typography from '../../../common/Typography';
+import TextField from '../../../common/TextField';
+import Button from '../../../common/Button';
+import signupValidate from '../../../../utils/application/SignupValidation';
+import SelectBox from '../../../application/SelectBox';
+import './index.css';
```

Comment:
add style import under images import as a type of organization for  your code. its a suggestion only.